documentation/pipeline.man

```diff
@@ -177,6 +177,10 @@ Argument is optional. Print the determined ra, de, and roll to \fIpath\fP. Defau
 \fB--compare-attitude\fP [\fIpath\fP]
 Argument is optional. Compare expected to actual attitude. Prints to \fIpath\fP. Defaults to stdout if \fIpath\fP is not given.
 
+.TP
+\fB--coverage-test\fP [\fIpath\fP]
+Argument is optional. Generate random attitudes for performance benchmark. Prints to \fIpath\fP. Defaults to stdout if \fIpath\fP is not given.
+
 .TP
 \fB--centroid-compare-threshold\fP \fIthreshold\fP
 Sets the threshold for the centroid comparing algorithms
```

Comment:
Also, about both command line options: To stay in line with the way all the other command-line options are named, any command line option whose only purpose is to change the way images are generated should be prefixed with `--generate`. For example, it would be more appropriate to use something like `--generate-random-attitude` and `--generate-time-based-seed` instead of `--coverage-test` and `--time-based-seed` for your command-line options.